Common/Gfx/Shape.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#ifndef RM_GFX_SHAPE_H_
+#define RM_GFX_SHAPE_H_
+#include <gdiplus.h>
+namespace Gfx {
+	struct Shape
```

Comment:
We don't indent within namespace blocks. Can you also add a newline before and after the `namespace` line?